week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -1,47 +1,82 @@
 // The below functions are syntactically correct but not outputting the right results.
 // Look at the tests and see how you can fix them.
 
+//main source - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators
+//https://www.w3schools.com/js/js_comparisons.asp
+
+//expected test("mood function works", mood() === "I am not happy");
+
 function mood() {
   let isHappy = true;
-
-  if (isHappy) {
+  if (!isHappy) {
```

Comment:
This works, however, you could also change the `isHappy` value on line 10 to `false`. In this way, the if-statement would continue to make sense to a fellow human reading this code.